Bottom: 33f5c851bf99db43056cacd37cccdb01dfe0971d
Top:    ca96f93494a442a8acc0029254b5f013dc5d40c7
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:27 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 41babc6..da0de99 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1612,6 +1612,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
    *
    */
 
+    fprintf (stderr, "Subtracting pin + via drills\n");
+    r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+    r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
   if (info.poly == NULL)
     {
       fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
